src/tools/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "tools"
+version = "0.0.0"
+authors = ["Alex Crichton <alex@alexcrichton.com>"]
```

Comment:
```suggestion
authors = ["The Rust Project Developers"]
```

nit: Should be?

Reply:
idk, I just copied it from the other `Cargo.toml` files 😉